source/frontend/src/features/properties/map/project/ViewSelector.tsx

```diff
@@ -38,7 +38,6 @@ export const ViewSelector = React.forwardRef<FormikProps<any>, IViewSelectorProp
         setProject={props.setProject}
         setContainerState={props.setContainerState}
         activeTab={props.activeTab}
-        isEditing={props.isEditing}
```

Comment:
seems unrelated?

Reply:
a little unrelated, but definitely not needed.